### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by jschueller**
cc @pcaclin

### CODE REVIEW

File: python/doc/user_manual/statistics_on_sample.rst [hunk 1]
```diff
@@ -129,6 +129,7 @@ Correlation analysis
     CorrelationAnalysis_PRCC
     CorrelationAnalysis_SRC
     CorrelationAnalysis_SRRC
+    CorrelationAnalysis_SignedSRC
```
Comment (sofianehaddad) [inline python/doc/user_manual/statistics_on_sample.rst]: My bad, I forgot about that

File: ChangeLog [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 ==== New classes ====
  * ParametricPointToFieldFunction
+ * OPTpp
```
Comment (sofianehaddad) [inline ChangeLog]: Maybe should be also mentioned on the website ?

**Comment by sofianehaddad** [comment]
Thanks for the swig typemaps

**Comment by regislebrun** [comment]
+1

**Comment by jschueller** [comment]
@sofianehaddad the new swig typemaps do not pass the tests, leaving for later

**Pull request merged**